reactivesocket-tck-drivers/build.gradle

```diff
@@ -0,0 +1,36 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.13'
+}
+
+apply plugin: 'application'
+apply plugin: 'java'
+
+mainClassName = "io.reactivesocket.tckdrivers.main.Main"
+
+jar {
+    manifest {
+        attributes "Main-Class": "$mainClassName"
+    }
+
+    from {
+        configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
+dependencies {
+
+    compile project(':reactivesocket-core')
+    compile project(':reactivesocket-client')
+    compile project(':reactivesocket-transport-tcp')
+    compile project(':reactivesocket-core')
+    compile 'io.netty:netty-all:4.1.3.Final'
```

Comment:
Remove this and rely on the transitive dependency from reactivesocket-transport-tcp